packages/app/components/Queue/TA/TAQueueDetailButtons.tsx

```diff
@@ -117,18 +118,27 @@ export default function TAQueueDetailButtons({
             await API.questions.notify(question.id);
           }}
         >
-          <Tooltip title="Remove From Queue">
-            <BannerDangerButton
-              shape="circle"
-              icon={<DeleteOutlined />}
-              data-cy="remove-from-queue"
-            />
+          <Tooltip
+            title={
+              isCheckedIn
+                ? "Remove From Queue"
+                : "You must check in to remove students from the queue"
+            }
+          >
+            <span>
+              {/* This span is a workaround for tooltip-on-disabled-button 
```

Comment:
Does this tooltip work properly? @NEUDitao and I were messing around with tooltips on disabled buttons by wrapping them in a span for adhoc queues and it was just acting up way too much

Reply:
It seemed to be working for me